python/tank/template.py

```diff
@@ -396,26 +398,24 @@ class TemplatePath(Template):
     """
     Class for templates for paths.
     """
-    def __init__(self, definition, keys, root_path, name=None):
+    def __init__(self, definition, keys, root_path, name=None, root_paths_all_os=None):
```

Comment:
agreed! I was trying to be very explicit about the fact that it was *all* platforms though - just to distinguish it from the `root_path` param. Any suggestions containing "all" or similar? Maybe `per_platform_paths`?